src/js/utils/phpDateTimeMomentFormat.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+(function(m) {
```

Comment:
do you think there is a way to implement it by exporting as a module instead of as an iife?